src/cript/nodes/primary_nodes/project.py

```diff
@@ -76,6 +80,37 @@ def __init__(
         self._json_attrs = replace(self._json_attrs, name=name, collections=collections, material=material)
         self.validate()
 
+    def validate(self):
+        # First validate like other nodes
+        super().validate()
+
+        # Check graph for orphraned nodes, that should be listed in project
+        # Project.materials should contain all material nodes
+        all_materials = self.find_children({"node": ["Material"]})
+        for material in all_materials:
+            if material not in self.materials:
+                raise CRIPTOrphranedMaterialError(material)
+
+        # Check graph for orphraned nodes, that should be listed in the experiments
+        all_experiments = self.find_children({"node": ["Experiment"]})
+        # All in the graph has to be in at least one experiment
+        node_types = ("Process", "Computation", "ComputationalProcess", "Data")
+        for node_type in node_types:
+            all_nodes = self.find_children({"node": [node_type]})
```

Comment:
does this make any sense?

I think this is saying `within process find process`